examples/ffi/e2e_kernel.py

```
@@ -0,0 +1,69 @@
+    Benchmark,
+    BenchmarkConfig,
+    TraceSet,
+    apply,
+    enable_apply,
+    disable_apply,
+)
+
+def main():
+    traceset_path = "Example-FlashInfer-Trace"
+    
+    print(f"Loading Example-FlashInfer-Trace")
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)